momento-sdk/src/intTest/java/momento/sdk/BaseStorageTestClass.java

```diff
@@ -0,0 +1,55 @@
+package momento.sdk;
+
+import java.time.Duration;
+import java.util.UUID;
+import momento.sdk.auth.CredentialProvider;
+import momento.sdk.config.StorageConfigurations;
+import momento.sdk.responses.storage.CreateStoreResponse;
+import momento.sdk.responses.storage.DeleteStoreResponse;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+
+public class BaseStorageTestClass {
+  protected static final Duration TEN_SECONDS = Duration.ofSeconds(10);
+  protected static CredentialProvider credentialProvider;
+  protected static PreviewStorageClient storageClient;
+  protected static String storeName;
+
+  @BeforeAll
+  static void beforeAll() {
+    credentialProvider = CredentialProvider.fromEnvVar("TEST_AUTH_TOKEN");
```

Comment:
This will need an update since your other pr got in first.

Reply:
I will rebase that on the cache rename once that is through. This was the first PR.